docs/source/raft_ann_benchmarks.md

```diff
@@ -0,0 +1,169 @@
+# RAFT ANN Benchmarks
+
+This project provides a benchmark program for various ANN search implementations. It's especially suitable for comparing GPU implementations as well as comparing GPU against CPU.
+
+## Installing the benchmarks
+
+You can easily install the benchmarks through conda with the following instructions:
```

Comment:
Maybe "The most easy way to install these benchmarks is through conda. We suggest using `mamba` as it generally leads to a faster install time:"